packages/rpc-client/package.json

```diff
@@ -41,6 +42,8 @@
         "@0x/types": "^3.1.1",
         "@0x/typescript-typings": "^5.0.1",
         "@0x/utils": "^5.1.1",
+        "@types/chai": "^4.2.8",
+        "@types/chai-as-promised": "^7.1.2",
```

Comment:
Types are typically `devDependencies`. What am I missing?

Reply:
Nothing, I missed this.